### PR TITLE
add HIL_ACTUATOR_CONTROLS mavlink message

### PR DESCRIPTION
**Pull request by bkueng**
This is the replacement message for the HIL_CONTROLS message.

I couldn't actually test if this compiles because I don't have ROS installed.

### CODE REVIEW

File: mavros/src/plugins/hil_actuator_controls.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+public:
+	HilActuatorControlsPlugin() : PluginBase(),
+		hil_actuator_controls_nh("~hil_actuator_controls")
+	{ }
+
+	void initialize(UAS &uas_)
+	{
+		PluginBase::initialize(uas_);
+
+		hil_actuator_controls_pub = hil_actuator_controls_nh.advertise<mavros_msgs::HilActuatorControls>("hil_actuator_controls", 10);
```
Comment (vooon) [inline mavros/src/plugins/hil_actuator_controls.cpp]: Do you really need that long topic name: */mavros***/hil_actuator_controls/hil_actuator_controls**

I thing you may use `~` in node handle init.

File: mavros_msgs/msg/HilActuatorControls.msg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# HilControls.msg
```
Comment (vooon) [inline mavros_msgs/msg/HilActuatorControls.msg]: Fix this. And add that file to CMakeLists.txt (CI failure).

**Comment by vooon** [comment]
Does that message exist in mavlink 2016.8.25? If not, next release scheduled on September 9.

**Comment by bkueng** [comment]
Thanks, I updated according to your feedback. The message was added to mavlink 2016.08.30, https://github.com/mavlink/mavlink/commit/218496ca5629839e2bf476dbdfdc4da870697cf7
Is this the reason for the CI error?
```
/home/travis/catkin_ws/src/mavros/mavros/src/plugins/hil_actuator_controls.cpp:54:97: error: ‘mavlink::common::msg::HIL_ACTUATOR_CONTROLS’ has not been declared
```

**Comment by vooon** [comment]
Yes, that message not exist - no class declaration.

**Comment by vooon** [comment]
Mavlink was released, CI passes. Merged.

**Pull request merged**